### PR TITLE
Payment templates

### PR DESCRIPTION
**Pull request by muzafarova**
Hotel only. 
Please take a look, if all ok, will continue to other domains

### CODE REVIEW

File: src/builtin-templates/hosted-payment-card-form.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    };
+
+    const optionsString = Object.keys(options)
+        .filter(key => options[key])
+        .map(key => `&${key}=${options[key]}`)
+        .join('');
+
+    const src = `${BASE_URL}?otp=${otp}${optionsString}`;
+
+    return html`
```
Comment (qubyte) [inline src/builtin-templates/hosted-payment-card-form.js]: Let's stick with vanilla DOM stuff for builtin templates now. In theory, it should work just fine to drop the `html` tag here and replace it with a `document.createElement('iframe')`...

File: src/render-summary.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export default class Summary {
 
         updateUi.call(this);
 
-        window.addEventListener('update', updateUi);
+        window.addEventListener('update', () => updateUi.call(this));
```
Comment (qubyte) [inline src/render-summary.js]: Looks like you need to rebase on master.

File: src/builtin-templates/hosted-payment-card-form.js (hunk 1 above)
Comment (muzafarova) [reply src/builtin-templates/hosted-payment-card-form.js]: ok!

File: src/render-summary.js (hunk 2 above)
Comment (muzafarova) [reply src/render-summary.js]: yup

**Pull request merged**